packages/nimble-components/src/theme-provider/index.ts

```diff
@@ -33,19 +54,44 @@ export const theme = DesignToken.create<Theme>({
  * @internal
  */
 export class ThemeProvider extends FoundationElement {
-    @attr({
-        attribute: 'direction'
-    })
+    @attr()
+    public override lang = '';
+
+    @attr()
     public direction: Direction = Direction.ltr;
 
-    @attr({
-        attribute: 'theme'
-    })
+    @attr()
     public theme: Theme = Theme.light;
 
+    public get validity(): ValidityObject {
```

Comment:
This will ultimately need to be exposed through Angular & Blazor, right? Is the plan to do that in a different PR, or should they be added to this one?

Reply:
I'll do it in a follow-up PR.